Fakebook.Profile/Fakebook.Profile.DataAccess/Services/IBlobService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Fakebook.Profile.DataAccess.Services
+{
+    /// <summary>
+    /// Interface for interacting with azure.
+    /// </summary>
+    public interface IBlobService
```

Comment:
semantically decouple the interface from azure blob storage, so remove "blob" and "azure" from this file. maybe IStorageService? we still need the container name parameter i guess which is sort of an implementation detail but it's fine.

document this method here too, including example values for params.